.github/workflows/e2e-on-demand.yaml

```
@@ -110,19 +114,58 @@ jobs:
+        uses: actions/setup-node@v4
+        with:
+          node-version: "22"
+
       - name: Create Kind cluster
         run: kind create cluster --name mcp-gateway --config config/kind/cluster-ci.yaml
 
       - name: Setup CI environment
         run: make ci-setup
 
+      - name: Setup Fallback LLM (Ollama)
+        if: (needs.check-permission.outputs.suite == 'gevals' || needs.check-permission.outputs.suite == 'full') && secrets.MODEL_KEY == ''
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)